addons/html_editor/static/src/wysiwyg.js

```diff
@@ -39,9 +39,16 @@ export function useWysiwyg(target, config = {}) {
     config.inlineComponentInfo = {
         app: comp.__owl__.app,
         env,
-    }
+    };
     const ref = typeof target === "string" ? useRef(target) : null;
-    const editor = new Editor(config, env.services);
+    const localOverlaContainerRef = useRef(config.localOverlayRefName);
```

Comment:
why not simply write on the config like this:
```js
config.getLocalOverlayContainer = ...;
```
?
Note that it modifies the config, so if you prefer, you could do something like this:
```js
const _config = Object.create(config);
_config.getLocalOverlayContainer = ...;
```
This has the advantage of not spreading the config, which may contain getters/setters, so spreading it would evaluate getters immediately, instead of when the recipients decides to use it

Reply:
haha another argument for not using getters/setters

Reply:
I've applied your suggestion